packages/manager/apps/pci-block-storage/src/components/Encryption.tsx

```diff
@@ -47,9 +47,14 @@ const Radio = ({
 interface Props {
   encryptionType: EncryptionType | null;
   onChange: (value: EncryptionType | null) => void;
+  texts: {
+    title: string;
+    description?: string;
+    badge?: string;
```

Comment:
I think you should either make it always new (check with design team for this), or use instead a "new" boolean property

Having a generic text badge with the color new doesn't seem like something that can be reused